IlmBase/HalfTest/testClassification.cpp

```diff
@@ -11,12 +11,14 @@
 #include "half.h"
 #include <iostream>
 #include <assert.h>
-
+#include <type_traits>
 
 using namespace std;
 
 namespace {
 
+static_assert(std::is_trivially_default_constructible<half>::value, "half is trivial and default constructible");
```

Comment:
is_trivially_default_constructible is only partially supported, and not supported in GCC 4.8.5, so this could be conditional. https://gcc.gnu.org/onlinedocs/gcc-4.8.3/libstdc++/manual/manual/status.html#status.iso.2011

Reply:
Perhaps hold this for the next version bump of OpenEXR, or guard with a gcc version check? vfxplatform has specified gcc 6.3.1 for a couple of years now.

Reply:
A lot of places are still using 4.8 / C++11.

Reply:
Noted! 4.8, grumble grumble

Reply:
Oh, I share your grumbles. I'm going to throw a party when I can move all of my packages to be gcc6/C++14 minimum. (And an even bigger one when it's all C++17.)

Reply:
And here I was being happy I remembered to use the c++11 syntax (as opposed to the _v variant) :)

lame. ok.

Reply:
Hasn't it been just recently decided by the TSC that OpenEXR requires a c++-11 conformant compiler to build (from now on)?

Reply:
yeah, well, how about "mostly" compliant? What we mostly decided to do is to start removing the need to keep c++98 branches of the code alive.